CMakeLists.txt

```diff
@@ -259,6 +260,10 @@ if(CMAKE_C_COMPILER_ID MATCHES "MSVC")
   )
 
   string(APPEND CMAKE_EXE_LINKER_FLAGS " /NXCOMPAT")
+  # Generate debug data
+  string(APPEND CMAKE_EXE_LINKER_FLAGS " /DEBUG:FASTLINK")
```

Comment:
Even tho this appears in the options passed to link.exe, it is actually coming from the defaulted value. It's not explicitly set because of things like this: https://devblogs.microsoft.com/cppblog/playground-games-and-turn-10-studios-see-18-2x-and-4-95x-link-time-improvements-respectively-on-visual-studio-2019/#comment-1554 where /DEBUG:full is preferable and will probably start defaulting back to that...

i.e. just pass `/DEBUG`

Reply:
Done